src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -113,7 +113,8 @@ public final class BlocklyJavascriptBridge {
 						if (c != null) {
 							selected.set(String.format("#%02x%02x%02x", c.getRed(), c.getGreen(), c.getBlue()));
 						}
-					}, null);
+						JColor.disposeDialog();

```

Comment:
I think less error-prone solution would be to hide colorChooser and instead provide public helper static method to open color chooser from JColor that would take care of dispose logic.

This way user does not need to be careful about this when using JColor's colorChooser